gorm/plugin/tracing/tracing_test.go

```
@@ -358,6 +348,63 @@ func TestMapDBSystem(t *testing.T) {
+		}
+		defer func() { // Close DB during cleanup
+			sqlDB, err := db.DB()
+			if err != nil {
+				t.Fatalf("failed to connect database: %v", err)
+			}
+			_ = sqlDB.Close()
+		}()
+
+		db = db.Debug() // Set to debug mode
+		tracer := tracetest.NewTracer()
+		err = db.Use(NewPlugin(
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)